app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -266,6 +268,39 @@ export default class Heatmap {
       `background-color: ${this.options.svgBackgroundColor}`
     );
 
+    const defs = this.svg.append("defs");
+
+    // Creates a color matrix that can recolor a black icon via a filter. See:
+    // https://semisignal.com/using-fecolormatrix-to-dynamically-recolor-icons-part-1-single-color-icons/
+    const generateColorMatrix = function(rgb) {
+      let rScaled = rgb[0] / 255.0;
+      let gScaled = rgb[1] / 255.0;
+      let bScaled = rgb[2] / 255.0;
+      return `0 0 0 0 ${rScaled}
+              0 0 0 0 ${gScaled}
+              0 0 0 0 ${bScaled}
+              0 0 0 1 0`;
+    };
+
+    // Convert hex string to rgb array
+    const hexToRgb = function(hex) {
+      var r = parseInt(hex.slice(1, 3), 16),
+        g = parseInt(hex.slice(3, 5), 16),
+        b = parseInt(hex.slice(5, 7), 16);
+      return [r, g, b];
+    };
+
+    // Create a blue color filter to match $primary-light.
+    defs
+      .append("filter")
+      .attr("id", "blue")
+      .append("feColorMatrix")
+      .attr("type", "matrix")
+      .attr(
+        "values",
+        generateColorMatrix(hexToRgb(this.options.colorHoverLink))
+      );
```

Comment:
Could we factor this out into a function like `addSvgColorFilter` and put it in `utils/d3/svg.js`?

You'd then call the function with `addSvgColorFilter(this.svg, "blue", this.options.colorHoverLink)`.